torch/_torch_docs.py

```diff
@@ -388,6 +401,35 @@ def parse_kwargs(desc):
             [ 3.,  6.]])
 """)
 
+add_docstr(torch.as_tensor,
+           r"""
+as_tensor(data, dtype=None, device=None, requires_grad=False) -> Tensor
+
```

Comment:
nit: do we have `requires_grad` here? Should we add it only when we decide on the semantics?

Reply:
good catch, I didn't update the docs.